api-admin/src/main/java/bio/terra/pearl/api/admin/service/enrollee/EnrolleeRelationExtService.java

```diff
@@ -44,4 +50,68 @@ public List<EnrolleeRelation> findRelationsForTargetEnrollee(
             .orElseThrow(() -> new NotFoundException("Enrollee not found"));
     return enrolleeRelationService.findByTargetEnrolleeIdWithEnrolleesAndFamily(enrollee.getId());
   }
+
+  // todo
+  @EnforcePortalStudyEnvPermission(permission = "???")
```

Comment:
I have no idea what to put on these methods (aside from `BASE`). I would like to make this a prototype_user only thing for now - but that's a role, not a permission. It could be cool to make a `can_use_beta_features` permission that this could sit under until our permissions are more mature and we can decide what to do with these.

Reply:
That's a great point, let's talk after standup.  I suspect what we want to do is replace the current `survey_response_edit` permission with a more general `participant_data_edit` permission.  And then this method will have that

How I'd assumed beta permissions would would is that you would annotate this feature with a specific permission that you create for the feature like "family_relation_edit", and then you'd add that permission to the "prototype_user" role.  That may be more complexity/trouble than these are worth though -- we might be better off as you said, adding a "beta_feature" permission.